Testing/functional/e2e/cypress/integration/services/medication.js

```diff
@@ -16,6 +23,47 @@ describe("Medication Service", () => {
             );
         });
     });
+    
+    it("Verify Get Medications List", () => {
+        const drugIdentifiersQueryString = 'drugIdentifiers=66999990&drugIdentifiers=02391724&drugIdentifiers=02212048&drugIdentifiers=02263238&drugIdentifiers=00496499&drugIdentifiers=02042258&drugIdentifiers=02240550&drugIdentifiers=01926799&drugIdentifiers=02237250&drugIdentifiers=02252716&drugIdentifiers=02263254&drugIdentifiers=02046121&drugIdentifiers=02271958&drugIdentifiers=02240606&drugIdentifiers=02247701&drugIdentifiers=02046148&drugIdentifiers=66999997&drugIdentifiers=00402753&drugIdentifiers=00029246';
+        cy.get("@config").then((config) => {
+            cy.log(`Medication Service Endpoint: ${config.serviceEndpoints.Medication}`);
+            cy.request({
+                url: `${config.serviceEndpoints.Medication}v1/api/Medication/?${drugIdentifiersQueryString}`,
+                followRedirect: false,
+                headers: {
+                    accept: "application/json",
+                },
+            }).should((response) => {
+                expect(response.status).to.eq(200);
+                expect(response.body).to.not.be.null;
+                cy.log(`response.body: ${JSON.stringify(response.body)}`);
```

Comment:
FYI:  You don't have to stringify instead you can simply log the object and inspect it in the console.

Reply:
> FYI: You don't have to stringify instead you can simply log the object and inspect it in the console.
@sslaws good to know, as I saw only [Object Objec] in the console so I though I needed the stringify.

Reply:
removed stringify